pkg/apimachinery/patch/patch.go

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  *
- * Copyright 2018 Red Hat, Inc.
+ * Copyright The KubeVirt Authors
```

Comment:
Is it okay to change copyright of an existing file?

Reply:
Ok, let me change it back you are right